can/io/blf.py

```diff
@@ -0,0 +1,263 @@
+"""
+Implements support for BLF (Binary Logging Format) which is a proprietary
+CAN log format from Vector Informatik GmbH.
+
+No offical specification is available. The implementation is based on the works
+by Toby Lorenz (https://bitbucket.org/tobylorenz/vector_blf).
+The file starts with a header. The rest is one or more "log containers"
+which consists of a header and some zlib compressed data, usually up to 128 kB
+of uncompressed data each. This data contains the actual CAN messages and other
+objects types.
+"""
```

Comment:
Actually on a similar note can you add a line to the `doc/history.rst#acknowledgements` to acknowledge both Toby Lorenz and your work for this.